### PR TITLE
# %ASA-3-713163: Remote user (session Id - id) has been terminated by the Firewall Server. Sprint 3

### LINKED ISSUE #70: Implementing Firewall log %ASA-3-713163: Remote user (session Id - id) has been terminated by there server

### PR DESCRIPTION
**Pull request by NazWright**

### CODE REVIEW

File: id_pkg/src/log_parse.py [hunk 1]
```diff
@@ -108,6 +108,8 @@ def handle_message(self, df, id):
             if m:
                 df.loc[id, 'Error'] = m.group(1)
 
+        # %ASA-3-713163: Remote user (session Id - id) has been terminated by the Firewall Server
```
Comment (claesmk) [inline id_pkg/src/log_parse.py]: I'm not clear on what these comments are for

**Review by claesmk — CHANGES_REQUESTED** [review]
It looks like you are trying to pull this into the main branch - you should instead use your team's branch - [ghost_ids](https://github.com/NCATComp410/comp410_spring_2021/tree/ghost_ids)

**Review by NazWright — COMMENTED** [review]
@claesmk  I have made the necessary changes for review

**Review by claesmk — APPROVED** [review]
Looks good

**Pull request merged**